### ISSUE TITLE
Ukraine support? When has code become political.

### ISSUE THREAD

**Issue by MarcGodard** [sveltejs/kit#4222]
### Describe the bug

You guys have no idea what is going on in Ukraine... why are you making political messages?

### Reproduction

Look at website.

### System Info

```shell
This is incredibly disappointing.
```

### Severity

serious, but I can work around it

### Additional Information

I avoid platforms and tools that are political. Again, you guys are in the fog of war and know nothing about what is going on there. Only propaganda from both sides.

**Comment by Conduitry** [comment]
We may not know exactly what's happening in Ukraine, but to say that there is overwhelming international consensus about enough of what's happening there to justify encouraging people to support a particular side would be a massive understatement.

**Comment by pngwn** [comment]
We often take a stand on political issues when we feel it is appropriate. If this is unacceptable to you, you are welcome to stop using svelte.

**Comment by MarcGodard** [comment]
@Conduitry How can you know anything when everything is being censored? Every opposing opinion is not allowed... I know people that live there, and what you are all being told are lies. Consensus is group think, and it is very bad. Am I the only one that questions anything now-a-days?

@pngwn but code isn't about politics and should never be. This pollutes politics, code, and truth. Only someone without the ability to critically think would think mixing this is good. Have you not noticed that everything that becomes politics becomes polluted? Are so many so blind to the powers behind propaganda and politics?

Like have people lost the ability to even think about what they are told? Has television programming actually programmed people into robots?

Don't get me wrong, putin is absolute evil, but so is the west... just look at yemem (bet you have no clue whats going on there) and tons and tons of other examples. The west is as much liars as putin is and you guys are all dumb enough to fall for it. My biggest concern is that when politics fall into code, you are pushing away people like me, and pushing away anyone that can critically think for themselves.

Until you have talked to people that live there (not people the news highlight) you really shouldn't think you know anything. Did you not think all the evil dictators pushed propaganda on their own people to push their own messages? Like Canada, peaceful protests are nazis, but actual nazis in ukraine are heroes?

Geez, try a little please... if you even did a bit of exploring the world would be so much better... can you even consider you are all part of the problem?

**Comment by mrkishi** [comment]
You are right: code isn't about politics. SvelteKit is MIT-licensed and no one could stop even war criminals from using it, should they wish.

But people aren't code, and they are also not expendable. I would love explore the world, as you say. I'd love to visit Ukraine and hear their stories directly from them. Unfortunately that's not sensible as the whole country is under attack right now.

There's no point in justifying war by pointing at other bad actors. "This is just as bad, why not blast it too?" isn't as convincing as you seem to think. In any case, why are you arguing with people "who can't think for themselves?" Why would you even _want_ to use code from supposedly irrational people?

You're free to use the project; you're free not to. The code is the same no matter your stance on any topic. Critically think about your priorities and stop conflating code with community.

**Comment by pngwn** [comment]
Open source is inherently political.

In this instance the maintainers have collectively decided to take a stand and show support to the folks in Ukraine. Just like we did previously with Black Lives Matter and when we joined the climate strike. You must not be familiar with the Svelte community and it's values if you think this is out of character. 

Could we be wrong? Absolutely. But we would rather use our critical thinking skills and make the best decision we can, with the information that we have, in order to take action that could have a positive impact in this scenario. We would rather take action and risk making a mistake than do nothing. Doing nothing is evil enough.

However, I'm not trying to convince you because this is a not a discussion. We are not for turning.
